scripts/tailwind_checker.ts

```diff
@@ -0,0 +1,131 @@
+import { walk } from "https://deno.land/std@0.201.0/fs/mod.ts";
+
+const green = "\x1b[32m";
+const reset = "\x1b[0m";
+
+const folderPaths = ["./components", "./sections", "./islands"];
```

Comment:
do we need to filter per folder? can't we just search for all `.tsx` except the ones that is listed at `.gitignore` (like node_modules)?